src/linelist.jl

```diff
@@ -1,132 +1,229 @@
-const numerals = ["I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX", "X"]
+import Base: (==), hash
+using CSV
 
 """
-    parse_species_code(code)
-
-Parse the "species code" as it is often specified in linelists and return a the "astronomy" 
-notation. 01.00 → "H_I", 02.01 → "He_II", 02.1000 → "He_II", 0608 → "CO_I", etc.  
+Represents an atom or molecule, irespective of its charge.
 """
-function parse_species_code(code::AbstractString)
-    toks = split(code, '.')
+struct Formula
+    atoms::Vector{String}
```

Comment:
As we previously discussed, I definitely think it would be useful to make the ``atoms`` field a static vector of 3 or 4 `UInt8`/`Int8` values. But I understand if you want to punt on that (for now)

Reply:
Yeah I'm going to add this to the TODO issue and I'd love to tackle it before 1.0, but I'd like to merge this PR with the current implementation.

Reply:
Sounds like a plan. 

If there's a pre-1.0 crunch , I would suggest that we at least convert from a `Vector{String}` to a `StaticVector{String}` (it bothers me a little that these datatypes are technically mutable)